pkgs/desktops/plasma-5/plasma-desktop/default.nix

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Kinda weird, but I won't judge.

Reply:
That was not on purpose :D
Thanks for catching it